WP-API/jwt-auth
# Fix manual token creation with plain permalinks

Uses the built-in `get_rest_url` API inside of `get_rest-uri` – this results in permalink-style-aware URI generation.

Fixes #4 

**To Test:**
Before – set a site to use ~default~ plain permalinks. Attempt to create a manual token. It should fail with a network error.
After – set a site to use ~default~ plain permalinks. Attempt to create a manual token. It should succeed.
